CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to segbo
+
+
+## Prerequisites
+
+Clone the repos
+```
+git clone --recurse-submodules https://github.com/cldf-datasets/segbo
```

Comment:
@J22Melody I think this is all you need to get submodules working on your end.

Reply:
I think it should point to https://github.com/segbo-db/segbo